src/Mod/Sketcher/Gui/Command.cpp

```diff
@@ -685,9 +684,9 @@ void CmdSketcherMapSketch::activated(int iMsg)
         bool bAttach = true;
         bool bCurIncompatible = false;
         // * find out the modes that are compatible with selection.
-        eMapMode curMapMode = eMapMode(sketch->MapMode.getValue());
+        const auto  curMapMode = eMapMode(sketch->MapMode.getValue());

```

Comment:
since the linter is disabled it is even more important to proof read the code before committing it, note the space that sneaked in.